src/photos/ducks/clustering/albums.js

```diff
@@ -204,7 +261,7 @@ export const albumsToClusterize = async (newPhotos, albums) => {
             )
             clusterAlbumsExist = true
             break
-          } else if (intersection(matchingAlbums, clusterAlbums).length > 1) {
+          } else if (intersection(matchingAlbums, clusterAlbums).length > 0) {
```

Comment:
Hm, I'm a bit worried that we missed this. Could we add a test to cover this?

Reply:
Hum yes, it would be better. However, I cannot test this function directly because of the problem we had with cozy-client-js (We couldn't import it with jest for obscure reason). So any idea how to test it?